scalameter-core/src/main/scala/org/scalameter/Measurer.scala

```diff
@@ -2,9 +2,14 @@ package org.scalameter
 
 
 
-import collection._
-import compat._
-import utils.withGCNotification
+import java.lang.reflect.Method
+import scala.collection._
+import scala.compat._
+import scala.runtime.BoxesRunTime
+import scala.util.matching.Regex
+import org.scalameter.execution.invocation.InvocationCountMatcher
```

Comment:
Yes, this is how it used to be organized in the past, but I don't think that was a good decision any more.
Code is often migrated between projects, and what used to be internal can become external at one point. And when names change, you have to refactor the imports anyway, at which point you hopefully reorder them as well.